Rubberduck.VBEEditor/ComManagement/TypeLibs/DocClassHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Text;
+
+namespace Rubberduck.VBEditor.ComManagement.TypeLibs
+{
+    /// <summary>
+    /// Extension to StringBuilder to allow adding text line by line.
+    /// </summary>
+    public class StringLineBuilder
+    {
+        private readonly StringBuilder _document = new StringBuilder();
+
+        public override string ToString() => _document.ToString();
+
+        public void AppendLine(string value = "")
+            => _document.Append(value + "\r\n");
+
+        public void AppendLineNoNullChars(string value)
+            => AppendLine(value.Replace("\0", string.Empty));
+    }
+
+    /// <summary>
+    /// A helper class for providing a static array of known VBA document class types
+    /// </summary>
+    public static class DocClassHelper
+    {
+        /// <summary>
+        /// An enumeration used for identifying the type of a VBA document class
+        /// </summary>
+        public enum DocClassType
```

Comment:
I would probably define the enum outside the class. That way you avoid having to write `DocClassHelper.DocClassType` as is the case further down to access the enum which also reads too much like an explicit interface implementation.

Reply:
done